sklearn/metrics/pairwise.py

```diff
@@ -653,13 +653,14 @@ def pairwise_distances_argmin_min(
         Y[argmin[i], :] is the row in Y that is closest to X[i, :].
 
     distances : ndarray
-        distances[i] is the distance between the i-th row in X and the
+        Distances[i] is the distance between the i-th row in X and the
         argmin[i]-th row in Y.
```

Comment:
Since it's the name of a variable, the uppercase is not really appropriate.
```suggestion
        The array of minimum distances. `distances[i]` is the distance between
        the i-th row in X and the argmin[i]-th row in Y.
```